examples/erc1155/lib.rs

```diff
@@ -229,36 +225,14 @@ mod erc1155 {
         token_id: TokenId,
     }
 
-    /// Represents an (Owner, Operator) pair, in which the operator is allowed to spend funds on
-    /// behalf of the operator.
-    #[derive(
-        Copy,
-        Clone,
-        Debug,
-        Ord,
-        PartialOrd,
-        Eq,
-        PartialEq,
-        Default,
-        PackedLayout,
-        SpreadLayout,
-        scale::Encode,
-        scale::Decode,
-    )]
-    #[cfg_attr(feature = "std", derive(scale_info::TypeInfo))]
-    struct Approval {
-        owner: AccountId,
-        operator: AccountId,
-    }
-
     /// An ERC-1155 contract.
     #[ink(storage)]
     #[derive(Default, SpreadAllocate)]
     pub struct Contract {
         /// Tracks the balances of accounts across the different tokens that they might be holding.
         balances: Mapping<(AccountId, TokenId), Balance>,
         /// Which accounts (called operators) have been approved to spend funds on behalf of an owner.
-        approvals: Mapping<Approval, ()>,
+        approvals: Mapping<(AccountId, AccountId), ()>,
```

Comment:
Please introduce a type `Owner` and a type `Operator` to make it clear which is which.